%YAML:1.0
---
image_width: 640
image_height: 360
intrinsic_left: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 4.6319424581582541e+02, 0., 3.1676090391833770e+02, 0.,
       4.6057998988141065e+02, 1.7397831981553443e+02, 0., 0., 1. ]
distCoeffs_left: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 3.1257611885074220e-02, 1.3395280591618429e-02,
       8.7824860098400311e-04, 8.6567631064158668e-04,
       -1.1465468683643715e-01 ]
intrinsic_right: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 4.6386409664026093e+02, 0., 3.2820581726567229e+02, 0.,
       4.6111156523537466e+02, 1.7689988006188253e+02, 0., 0., 1. ]
distCoeffs_right: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 2.5918770579236791e-02, 9.2111517300168845e-03,
       -4.6266245635179790e-04, 2.3977761004697171e-03,
       -7.5486110482767182e-02 ]
R: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9994907716543291e-01, -2.7315731920135797e-03,
       9.7150184712020658e-03, 2.6883763611468877e-03,
       9.9998645522460217e-01, 4.4566803649804913e-03,
       -9.7270606320692364e-03, -4.4303357921774700e-03,
       9.9994287657657155e-01 ]
T: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ -1.5151755817346250e+02, 2.8310737178178041e-01,
       -9.8793665109769002e-01 ]
R_L: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9985763176584852e-01, -4.6288074078050315e-03,
       1.6226223885127539e-02, 4.5927411374776109e-03,
       9.9998690111734223e-01, 2.2592747904558296e-03,
       -1.6236469087611227e-02, -2.1844302955502785e-03,
       9.9986579366225492e-01 ]
R_R: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 9.9997699817206342e-01, -1.8684359965105164e-03,
       6.5201283511817192e-03, 1.8829187469448024e-03,
       9.9999577240167037e-01, -2.2158058982944167e-03,
       -6.5159606951963487e-03, 2.2280318026133535e-03,
       9.9997628878414180e-01 ]
P1: !!opencv-matrix
   rows: 3
   cols: 4
   dt: d
   data: [ 4.8090766123277677e+02, 0., 3.1608622360229492e+02, 0., 0.,
       4.8090766123277677e+02, 1.7540640449523926e+02, 0., 0., 0., 1.,
       0. ]
P2: !!opencv-matrix
   rows: 3
   cols: 4
   dt: d
   data: [ 4.8090766123277677e+02, 0., 3.1608622360229492e+02,
       -7.2867630625602847e+04, 0., 4.8090766123277677e+02,
       1.7540640449523926e+02, 0., 0., 0., 1., 0. ]
Q: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [ 1., 0., 0., -3.1608622360229492e+02, 0., 1., 0.,
       -1.7540640449523926e+02, 0., 0., 0., 4.8090766123277677e+02, 0.,
       0., 6.5997433579760793e-03, 0. ]
validROIL: [ 1, 0, 639, 360 ]
validROIR: [ 0, 0, 640, 360 ]
